wiki/Guides/Videos_from_YouTube/en.md

```diff
@@ -0,0 +1,29 @@
+---
+tags:
+  - how to download videos from YouTube
+  - how to use YouTube videos
+---
+
+# Videos from YouTube
+
+[YouTube](https://youtube.com) is a very convenient resource. If a song has a music video, it will most likely be on YouTube, and probably compressed at that golden point where the video looks decent but will not be a pain to download.
+
+This tutorial is intended for those who **can not** find the video elsewhere but YouTube.
```

Comment:
not sure if this line is needed

Reply:
I think we want less videos sourced from yt and we have this guide as a last resort, but I'm not sure.